pages/api/auth/[...nextauth].ts

```diff
@@ -86,8 +87,9 @@ export const authOptions = {
 				const updateObj: DbUser = createUserUpdateObj(user, account, profile, dbUser);
 				await updateUser(dbUser.id!, updateObj).catch(err => {
 					console.error("[signIn] Count not update user in database", err);
+					rudderStackEvents.track("absent", uuidv4(), "auth", {type: "login", ...updateObj, newAuth: !existingAuth, eventtStatusFlag: 1 });
 				})
-				rudderStackEvents.track(dbUser.id!.toString(), uuidv4(), "login", { ...updateObj, newAuth: !existingAuth });
+				rudderStackEvents.track(dbUser.id!.toString(), uuidv4(), "auth", {type: "login", ...updateObj, newAuth: !existingAuth, eventtStatusFlag: 0 }); //TODO: Get the anonymoudId from the client session so that the random generated anonymoudId doesn't create noise.;
```

Comment:
```suggestion
				rudderStackEvents.track(dbUser.id!.toString(), uuidv4(), "auth", {type: "login", ...updateObj, newAuth: !existingAuth, eventStatusFlag: 0 }); //TODO: Get the anonymoudId from the client session so that the random generated anonymoudId doesn't create noise.;
```